src/components/MultiSelect/multiSelectChangelogData.ts

```diff
@@ -9,6 +9,15 @@
 import { ChangelogData } from "../../utils/ComponentChangelogTable";
 
 export const changelogData: ChangelogData[] = [
+  {
```

Comment:
I added this second object rather than combining this and the one before because they have different "types", but can adjust if wanted.

Reply:
You can merge them together and add both types in the `affects` array.